README.md

```diff
@@ -112,7 +112,7 @@ For more examples check out our [LLM Inference Guide](https://docs.openvino.ai/2
 optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format fp16 MiniCPM-V-2_6
 
 #(Recommended) Same as above but with compression: language model is compressed to int4, other model components are compressed to int8
-optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format int4 MiniCPM-V-2_6
+optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format int4 --dataset contextual MiniCPM-V-2_6 
```

Comment:
```suggestion
optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format int4 --dataset contextual --num-samples 1 MiniCPM-V-2_6 
```

This is faster and more memory efficient. Since no data-aware method is selected, dataset size won't have effect on the resulting model.

Btw, I plan to publish a fix to optimum-intel some time today.